### ISSUE TITLE
Validation accuracy of 31.25%?

### ISSUE THREAD

**Issue by mleming** [madnavs/Multivariate-Classification#1]
Hello, when I ran the Keras.py script it had a validation accuracy of 31%, which doesn't seem too great.

**Comment by azmiara** [comment]
Hello Naveen,
**It's not an issue but more of a doubt regarding the dataset.**
Greetings of the day!I have few doubts with your dataset .It will be great if you could help me solve this.
I would like to know what are the features mentioned in your dataset like vx,vy,vz and others. As it will give me a good understanding and I would like to take your code as a reference.
Request for your help.
